boards/arm/nucleo_l031k6/doc/index.rst

```diff
@@ -0,0 +1,169 @@
+.. _nucleo_l031k6_board:
+
+ST Nucleo L031K6
+################
+
+Overview
+********
+The STM32 Nucleo-32 development board with STM32L031K6 MCU, supports Arduino Nano V3 connectivity.
+
+The STM32 Nucleo board provides an affordable, and flexible way for users to try out new concepts,
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption, and features.
+
+The Arduino* Nano V3 connectivity support allow easy functionality
+expansion of the STM32 Nucleo open development platform with a wide choice of
+specialized shields.
+
+The STM32 Nucleo board integrates the ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Nucleo board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/nucleo_l031k6.jpg
+   :width: 426px
+   :height: 319px
+   :align: center
+   :alt: Nucleo L031K6
+
+More information about the board can be found at the `Nucleo L031K6 website`_.
+
+Hardware
+********
+Nucleo L031K6 provides the following hardware components:
+
+- STM32 microcontroller in LQFP32 package
+- Extension resource:
+
+  - Arduino* Nano V3 connectivity
+
+- ARM* mbed*
```

Comment:
Why is this in the list of hardware components?

Reply:
@sdaglish can you fix this ?

Reply:
I will look into fixing this issue, but I still think it would be better if all the STM32L0 boards had the same documentation. Therefore, I'll create an issue related to this so that all other boards get changed consistently.

Reply:
Reference to `Arm* mbed*`.
This is now inline with ST Nucleo L432KC documentation for Hardware.